common/index.html

```diff
@@ -176,8 +195,27 @@ <h2>Sample 2: Save to disk (resize on server)</h2>
         imageManagerDeleteURL: "/delete_image",
         imageManagerDeleteMethod: "POST",
         events: {
+        // Catch files removal from the files manager popup.
+       'filesManager.removed': function (link) {
+        $.ajax({
+          // Request method.
+          method: "POST",
+          // Request URL.
+          url: "/delete_file",
+          // Request params.
+          data: {
+            src: link
+          }
+        })
+        .done (function (data) {
+          console.log ('file was deleted from files manager popup');
+        })
+        .fail (function (err) {
+          console.log ('files manager popup delete problem: ' + JSON.stringify(err));
+        })
+      },
       // Catch video removal from the editor.
-      'froalaEditor.video.removed': function ($vid) {
```

Comment:
@singhmonaaccolite why did we remove this ? Was it not working before?

Reply:
It was not working before as the froalaEditor.video.removed is not an event.